src/BugsnagUnity/BugsnagSettingsObject.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using BugsnagUnity;
+using BugsnagUnity.Payload;
+using System;
+
+namespace BugsnagUnity.Editor
+{
+    public class BugsnagSettingsObject : ScriptableObject
+    {
+
+        public bool AutoDetectErrors = true;
+        public bool AutoTrackSessions = true;
+        public string ApiKey;
+        public string AppType;
+        public ulong AppHangThresholdMillis;
+        public string AppVersion;
+        public string BundleVersion;
+        public LogType BreadcrumbLogLevel = LogType.Log;
+        public string Context;
+        public string[] DiscardClasses;
+        public string[] EnabledReleaseStages;
+        public EnabledErrorTypes EnabledErrorTypes = new EnabledErrorTypes();
+        public EditorBreadcrumbTypes EnabledBreadcrumbTypes = new EditorBreadcrumbTypes();
+        public long LaunchDurationMillis = 5000;
+        public int MaximumBreadcrumbs = 25;
+        public LogType NotifyLogLevel = LogType.Exception;
+        public string NotifyEndpoint;
+        public string SessionEndpoint;
+        public string[] RedactedKeys = new string[] { "password" };
+        public string ReleaseStage = "production";
+        public bool ReportUncaughtExceptionsAsHandled = true;
+        public bool SendLaunchCrashesSynchronously = true;
+        public double SecondsPerUniqueLog = 5;
```

Comment:
Similar to other comment `double` seems overkill!  Also in .net you'd normally use a `decimal` for this kind of thing.